.changeset/wild-goats-march.md

```diff
@@ -0,0 +1,6 @@
+---
+'@itwin/itwinui-react': minor
+'@itwin/itwinui-css': minor
+---
+
+Added a new FileUploadCard component which serves as a default GUI for when a single file is uploaded using FileUpload
```

Comment:
Update description with usage example.

Reply:
Updated. I wasn't sure how in-depth to get with the usage examples, but I wrote a couple